app/adapters/alma_adapter/availability_status.rb

```diff
@@ -10,6 +10,60 @@ def initialize(bib:)
       @bib = bib
     end
 
+    # Returns availability information for each of the holdings in the Bib record.
+    def bib_availability
+      availability = {}
+      holdings.each do |holding|
+        status = holding_status(holding)
+        availability[status[:id]] = status
+      end
+      availability
+    end
+
+    # TODO: handle these properties
+    #   "copy_number": item["copy_number"],
+    #   "patron_group_charged": nil,    # TODO: nil
+    #   "status": "???",                # TODO: "Not Charged"
+    # TODO: Confirm that we need all of these values with the Alma implementation.
+    # TODO: Find out if we can get copy_number from Solr (i.e. without hitting ExLibris' API)
+    def holding_status(holding)
+      status = if holding["holding_id"]
+                 {
+                   on_reserve: "N",
+                   location: holding["location_code"],
+                   label: holding["location"],
+                   status_label: holding["availability"],
+                   more_items: (holding["total_items"] || "").to_i > 1,
+                   holding_type: "physical",
+                   id: holding["holding_id"]
+                 }
+               elsif holding["portfolio_pid"]
+                 # This kind of data is new with Alma.
+                 status = {
```

Comment:
issue: we shouldn't need `status=` if we have already set `status=` on line 30.

Reply:
Good catch. Fixed.